theurgy.lic

```diff
@@ -62,6 +62,69 @@ def clean_altar
   put_in_container($settings.water_holder)
 end
 
+def prayer_bead
+  return unless DRSkill.getrank('Mechanical Lore') >= 140
+
+  fput "stow #{GameObj.right_hand.noun}" if GameObj.right_hand.name != 'Empty'
+  fput "stow #{GameObj.left_hand.noun}" if GameObj.left_hand.name != 'Empty'
+  get_from_container('prayer chain')
+  chain_contents = DRC.bput("look at my chain", /^Strung on to the prayer bead chain you see .*\./, 'There are currently no beads on it.')
+  bead_count = DRC.list_to_nouns(chain_contents.match(/^Strung on to the prayer bead chain you see (.*)\./).to_a[1])
+  return unless bead_count.count < 10 
+  put_in_container('prayer chain')
+  DRC.forage? 'limb'
+  get_from_container($settings.water_holder)
+  fput("sprinkle #{$settings.water_holder} on my limb")
+  put_in_container($settings.water_holder)
+  if 'interrupt your research' == DRC.bput('prep bless', 'Bless spell', 'interrupt your research')
```

Comment:
I was thinking about checking the % and deciding to interrupt or wait it out.  I personally prefer the  interrupt in most cases.  I was thinking 60% could be the threshold